bridge/r2.js

```diff
@@ -0,0 +1,169 @@
+// @ts-check
+
+class R2ObjectBody extends Response {
+  /** @type {string?} */
+  key;
+
+  /** @type {string?} */
+  version;
+
+  /** @type {number?} */
+  size;
+
+  /** @type {string?} */
+  etag;
+
+  /** @type {string?} */
+  httpEtag;
+
+  /** @type {import("@cloudflare/workers-types").R2Checksums?}*/
+  checksums;
+
+  /** @type {Date?} */
+  uploaded;
+
+  /** @type {import("@cloudflare/workers-types").R2HTTPMetadata?}*/
+  httpMetadata;
+
+  /** @type {Record<string, string>?} */
+  customMetadata;
+
+  /** @type {import("@cloudflare/workers-types").R2Range?}*/
+  range;
+
+  /**
+   * @param {BodyInit | null} body
+   * @param {ResponseInit?} init
+   * @param {string?} key
+   * @param {string?} version
+   * @param {number?} size
+   * @param {string?} etag
+   * @param {string?} httpEtag
+   * @param {import("@cloudflare/workers-types").R2Checksums?} checksums
+   * @param {Date?} uploaded
+   * @param {import("@cloudflare/workers-types").R2HTTPMetadata?} httpMetadata
+   * @param {Record<string, string>?} customMetadata
+   * @param {import("@cloudflare/workers-types").R2Range?} range
+   */
+  constructor(
+    body,
+    init,
+    key,
+    version,
+    size,
+    etag,
+    httpEtag,
+    checksums,
+    uploaded,
+    httpMetadata,
+    customMetadata,
+    range
+  ) {
+    super(body, init ?? undefined);
+
+    this.key = key;
+    this.version = version;
+    this.size = size;
+    this.etag = etag;
+    this.httpEtag = httpEtag;
+    this.checksums = checksums;
+    this.uploaded = uploaded;
+    this.httpMetadata = httpMetadata;
+    this.customMetadata = customMetadata;
+    this.range = range;
+  }
+}
+
+// R2Namespace
+export class R2Bridge {
+  #wranglerOrigin;
+  #bindingName;
+
+  /**
+   * @param {string} wranglerOrigin
+   * @param {string} bindingName
+   */
+  constructor(wranglerOrigin, bindingName) {
+    this.#wranglerOrigin = wranglerOrigin;
+    this.#bindingName = bindingName;
+  }
+
+  /** @param {import("@cloudflare/workers-types").R2ListOptions} [options] */
+  async list(options) {
+    const url = new URL(this.#wranglerOrigin);
+    url.pathname = `/r2_list/${this.#bindingName}`;
+
+    const req = new Request(url, { method: "GET" });
+    if (options) req.headers.set("CF-R2-OPTIONS", JSON.stringify(options));
+
+    return await fetch(req);
```

Comment:
```suggestion
    const res = await fetch(req);
    return res;
```

Align code styles. 💅🏻